modules/snooker/src/main/ts/ephox/snooker/api/CopyCols.ts

```diff
@@ -33,7 +33,7 @@ const generateColGroup = (house: Warehouse, minColRange: number, maxColRange: nu
 
 const generateRows = (house: Warehouse, minColRange: number, maxColRange: number): SugarElement<HTMLTableRowElement>[] =>
   Arr.map(house.all, (row) => {
-    const cellsToCopy = Arr.filter(row.cells, (cell) => cell.column >= minColRange && cell.column < maxColRange);
+    const cellsToCopy = Arr.filter(row.cells, (cell) => (cell.column + cell.colspan > minColRange) && cell.column < maxColRange);
```

Comment:
This doesn't fix the issue... I've pulled down the changes and the content on the clipboard is still an empty row when selecting a column that has a row with a colspan.

Reply:
Additionally you've changed `>=` -> `>` which is going to definitely cause issues as it's meant to include the minimum value.

Reply:
Did some more debugging as it looked like the tests should have caught this and it seems the reason this isn't working is because `tsc` failed to recompile snooker sorry Hannes 😞

That said though, the `>` change is a little confusing and the parenthesis is in quite an odd position. So maybe something like this instead?
```suggestion
    const cellsToCopy = Arr.filter(row.cells, (cell) => {
      const endCol = cell.column + cell.colspan - 1;
      const startCol = cell.column;
      return endCol >= minColRange && startCol < maxColRange;
    });
```
This then keeps it more consistent to how it's done in `CopySelected.ts` as well. We could also abstract this out into a separate function to share between `generateRows` and `generateColGroup`.